doc/reference_en.html

```diff
@@ -2022,6 +2022,25 @@ <h3 class="not-for-contents">1. Why not to use systems like map-reduce?</h3>
 If you have an ORDER BY clause with some small LIMIT after a GROUP BY, then ORDER BY will not consume significant amount of memory. But if no LIMIT is provided, don't forget to turn on external sorting (%%max_bytes_before_external_sort%%).
 
 
+<h5>LIMIT N BY modifier</h5>
+
+LIMIT %%N%% BY %%COLUMN%% allows you to restrict top %%N%% rows per each group of %%COLUMN%%
```

Comment:
No mention about that key for LIMIT BY could contain arbitary number of columns and expressions.